src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/CreateApprenticeshipRequestValidator.cs

```diff
@@ -1,12 +1,74 @@
-﻿using FluentValidation;
+﻿using System;
+using System.Text.RegularExpressions;
+using FluentValidation;
+using static Esfa.Vacancy.Domain.Validation.ErrorCodes.CreateApprenticeship;
+
 
 namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship
 {
     public class CreateApprenticeshipRequestValidator : AbstractValidator<CreateApprenticeshipRequest>
     {
+        public const string RegexFreeTextWhitelist = @"^[a-zA-Z0-9\u0080-\uFFA7?$@#()""'!,+\-=_:;.&€£*%\s\/\[\]]+$";
+        public const string RegexHtmlFreeTextWhitelist = @"^[a-zA-Z0-9\u0080-\uFFA7?$@#()""'!,+\-=_:;.&€£*%\s\/<>\[\]]+$";
+        public const string RegexScriptsBlacklist = @"<\s*s\s*c\s*r\s*i\s*p\s*t\s*[^>]*\s*[^>]*\s*[^>]*>";
+        public const string RegexInputsBlacklist = @"<\s*i\s*n\s*p\s*u\s*t\s*[^>]*\s*[^>]*\s*[^>]*>";
+        public const string RegexObjectsBlacklist = @"<\s*o\s*b\s*j\s*e\s*c\s*t\s*[^>]*\s*[^>]*\s*[^>]*>";
+
+        private const string TitleApprentice = "apprentice";
+        private const int TitleMaximumLength = 100;
+        private const int ShortDescriptionMaximumLength = 350;
+
         public CreateApprenticeshipRequestValidator()
         {
-            RuleFor(request => request.Title).NotEmpty();
+            RuleFor(request => request.Title)
```

Comment:
We want a way to split these rules into separate files to manage the code a bit better and also avoid merge conflicts once we get into the subsequent stories.